slice/IceGrid/Admin.ice

```diff
@@ -625,160 +619,143 @@ module IceGrid
+        /// Notifies the observer that an application was removed.
         /// @param serial The new serial number of the registry database.
         /// @param name The name of the application that was removed.
         void applicationRemoved(int serial, string name);
 
-        /// The <code>applicationUpdated</code> operation is called to notify an observer that an application was
-        /// updated.
+        /// Notifies the observer that an application was updated.
         /// @param serial The new serial number of the registry database.
         /// @param desc The descriptor of the update.
         void applicationUpdated(int serial, ApplicationUpdateInfo desc);
     }
 
-    /// This interface allows applications to monitor the state of object adapters that are registered with IceGrid.
+    /// Monitors dynamically-registered object adapters.
     interface AdapterObserver
     {
-        /// <code>adapterInit</code> is called after registration of an observer to indicate the state of the registry.
+        /// Provides the initial list of dynamically registered adapters to the observer.
         /// @param adpts The adapters that were dynamically registered with the registry (not through the deployment
         /// mechanism).
         void adapterInit(AdapterInfoSeq adpts);
 
-        /// The <code>adapterAdded</code> operation is called to notify an observer when a dynamically-registered
-        /// adapter was added.
+        /// Notifies the observer that a dynamically-registered adapter was added.
         /// @param info The details of the new adapter.
         void adapterAdded(AdapterInfo info);
 
-        /// The adapterUpdated operation is called to notify an observer when a dynamically-registered adapter was
-        /// updated.
+        // Notifies the observer that a dynamically-registered adapter was updated.
         /// @param info The details of the updated adapter.
         void adapterUpdated(AdapterInfo info);
 
-        /// The adapterRemoved operation is called to notify an observer when a dynamically-registered adapter was
-        /// removed.
+        /// Notifies the observer that a dynamically-registered adapter was removed.
         /// @param id The ID of the removed adapter.
         void adapterRemoved(string id);
     }
 
-    /// This interface allows applications to monitor IceGrid well-known objects.
+    /// Monitors well-known objects that are added, updated or removed using {@link Admin}.
     interface ObjectObserver
     {
-        /// <code>objectInit</code> is called after the registration of an observer to indicate the state of the
-        /// registry.
-        /// @param objects The objects registered with the {@link Admin} interface (not through the deployment
-        /// mechanism).
+        /// Provides the initial list of well-known objects to the observer.
+        /// @param objects The well-known objects registered using {@link Admin} (not through the deployment mechanism).
```

Comment:
I think we should move `not through the deployment mechanism` to the interface description. My understanding this is true for all methods not just init.